hypervisor/dm/io_req.c

```diff
@@ -644,6 +643,8 @@ int32_t register_mmio_emulation_handler(struct acrn_vm *vm,
 				status = 0;
 			}
 		}
+	} else {
```

Comment:
Do we have a case hitting here?

Reply:
I guess not, and we can actually remove the check. Not?

Reply:
yes, will remove else , and remove "if (is_prelaunched_vm(vm) || (vm->state != VM_STARTED)) {",
change this api to void type